### PR TITLE
Add Jupyter Health Hubs

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://github.com/2i2c-org/infrastructure/issues/3796

### CODE REVIEW

File: config/clusters/jupyter-health/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: jupyter-health
+provider: aws
```
Comment (consideRatio) [inline config/clusters/jupyter-health/cluster.yaml:2]: cluster.yaml and this comment has been the goto place to figureout where a cluster is available - consistent across all current cluster.yaml files

```suggestion
provider: aws # https://2i2c.awsapps.com/start/#/
```

File: config/clusters/jupyter-health/common.values.yaml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                runAsUser: 0
+              volumeMounts:
+                - name: home
+                  mountPath: /home/jovyan
+                  subPath: "{username}"
+                # mounted without readonly attribute here,
+                # so we can chown it appropriately
+                - name: home
+                  mountPath: /home/jovyan/shared
+                  subPath: _shared
```
Comment (consideRatio) [inline config/clusters/jupyter-health/common.values.yaml:82]: We are already providing an init container by default to do this - aren't we?

**Review by consideRatio — COMMENTED** [review]
I left a comment about a init_container override that I think isn't relevant to specify, but otherwise this looks great.

**Comment by consideRatio** [comment]
Fixed a merge conflict

File: config/clusters/jupyter-health/common.values.yaml (hunk 2 above)
Comment (yuvipanda) [reply config/clusters/jupyter-health/common.values.yaml]: Yep! Another reason to move away from copy pasting hubs! This was leftover from copy pasting from nasa-veda

**Comment by yuvipanda** [comment]
CI caught that I hadn't added the `paid_by_us` field. Thanks for working on that, @consideRatio.

**Pull request merged**